config/initializers/doorkeeper_openid_connect.rb

```diff
@@ -27,12 +27,36 @@
   end
 
   claims do
-    claim :preferred_username, :scope => :openid do |resource_owner, _scopes, _access_token|
+    claim :preferred_username, :response => [:id_token, :user_info] do |resource_owner, _scopes, _access_token|
       resource_owner.display_name
     end
 
     claim :email, :scope => :read_email, :response => [:id_token, :user_info] do |resource_owner, _scopes, _access_token|
       resource_owner.email
     end
+
+    claim :email_verified, :scope => :read_email, :response => [:id_token, :user_info] do |resource_owner, _scopes, _access_token|
+      resource_owner.email_valid
+    end
+
+    claim :profile do |resource_owner, _scopes, _access_token|
+      "https://www.openstreetmap.org/user/#{resource_owner.display_name}"
+    end
+
+    claim :description do |resource_owner, _scopes, _access_token|
```

Comment:
I'm not sure how useful this, or any of the other custom claims you've added are given they're all OSM specific.

Reply:
Ok, so maybe it is best to remove the extra claims, and leave at minimum required.